### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by noelutz**
Update install instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ Note that you need a **recent** version of Node because we use new ES6 features.
 
 ```
 $ npm install -g gulp-cli
-$ (cd runtime && npm install)
+$ npm install -g bower
 $ (cd tracelib && npm install)
+$ (cd strategizer && npm install)
```
Comment (mmandlis) [inline README.md]: nit - any reason for the particular order? if not, maybe alphabetize: runtime, strategy, strategy-explorer, tracelib?

Comment (noelutz) [reply README.md]: I assumed things that depended on tracelib need to be installed after tracelib. Is that not the case?

Comment (mmandlis) [reply README.md]: IIUC 
(cd tracelib && npm install)
doesn't install tracelib, but rather locally installs the dependencies of the tracelib library, listed in the package.json file. so i don't think order matters (in fact we used to have runtime/ before tracelib/ and it worked)

Comment (smalls) [reply README.md]: To run the demo I think I had to run gulp webpack as well. Should we include that in the list of instructions?

**Pull request merged**